AutomationHelpers/scripts/install.ps1

```diff
@@ -29,4 +24,4 @@ $symbolString = $symbols -join ';'
 Write-Host "Replace with: $symbolString"
 $defineConstants.Value = $symbolString
```

Comment:
As mentioned before, this will break with project upgrades.
It would be great if we could dynamically create the compilation constant, i.e. by assigning a macro instead of a constant. Something like the following (if this works):
`$defineConstants.Value = RX + $(RanorexVersion).ToString(2)`

Reply:
> As mentioned before, this will break with project upgrades.
> It would be great if we could dynamically create the compilation constant, i.e. by assigning a macro instead of a constant. Something like the following (if this works):
> `$defineConstants.Value = RX + $(RanorexVersion).ToString(2)`

Hi @ahoisl 
I hope that should be fine for now and we move this to the end of the sprint, will take up if time permits. Thank you for your comments.

Reply:
OK, but then let's still leave it open like it is right now

Reply:
Hi @ahoisl
Ranorex Script Runner passes the Ranorex Version as PS variable. Additionally, I had added the check to ignore the non-helpers project and bumped up the version number. The additional check for non-helper projects will ensure that the script inserts the Rx-Version for Helper projects.

Reply:
So, if you just use the nuget package in any project, the RX... compilation constants will not be added? Please confirm, but I think it's alright

Reply:
Yes, @ahoisl that's the behaviour.